doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+[id='migration-tool-niogit-7.4-proc_{context}']
+
+= Using the migration tool on a `.niogit` directory
+
+The `.niogit` directory of your {CENTRAL} distribution contains all {CENTRAL} data. You can use the {CENTRAL} migration tool on your `.niogit` directory to prepare project data and system configurations for migration to {PRODUCT} 7.5.
+
+
+.Prerequisites
+* {PRODUCT} 7.5 is installed.
+* {PRODUCT} 7.4 in installed and {CENTRAL} is not running on {PRODUCT} 7.4.
+* Java 8 is installed and is available on the path where the project migration tool will be installed.
+
+.Procedure
+
+. Locate the `bin/.niogit` directory in your {PRODUCT} 7.4 installation and locate the same directory in your new {PRODUCT} 7.5 installation.
+
++
+--
+Example:
+
+
+[source,subs="attributes+"]
+----
+$ ~/${PRODUCT_INIT_CAP}_7.4_HOME/bin/.niogit
+----
+
+[source,subs="attributes+"]
+----
+$ ~/${PRODUCT_INIT_CAP}_7.5_HOME/bin/.niogit
+----
+--
+. In your command terminal, copy the `.niogit` directory from your current {CENTRAL} distribution for {PRODUCT} 7.4 to the `bin` folder of the new {PRODUCT} 7.5 installation directory:
++
+--
+
+[source,subs="attributes+"]
+----
+cp -r /${PRODUCT_INIT_CAP}_7.4_HOME/bin/.niogit  /${PRODUCT_INIT_CAP}_7.5_HOME/bin/
+----
+
+The copied `.niogit` folder is the directory that the migration tool will prepare for {PRODUCT} 7.5.
+. The original `.niogit` folder remains in your {PRODUCT} 7.4 installation and will not be modified by the migration tool in this procedure.
+--
+. In the {PRODUCT} 7.5. `bin` directory, delete the `.index` directory. You must delete this folder because {PRODUCT} 7.5 comes with certain updates that are not compatible with {PRODUCT} {PRODUCT_VERSION}. The `bin` directory will be re-indexed after the migration when you start {PRODUCT} 7.5.
```

Comment:
With the current values, the phrase here will read "because RHPAM 7.5 comes with certain updates that are not compatible with RHPAM 7.5", so you probably meant something else?

Reply:
Again, this is clear in the rendered doc:
In the Red Hat Process Automation Manager 7.5. bin directory, delete the .index directory. You must delete this folder because Red Hat Process Automation Manager 7.5 comes with certain updates that are not compatible with Red Hat Process Automation Manager 7.4. The bin directory will be re-indexed after the migration when you start Red Hat Process Automation Manager 7.5.

Reply:
After a rebase, however, {PRODUCT_VERSION} will become 7.5. I would suggest hardcoding 7.4 here.

Reply:
Well, this is for 7.4 only, but I have hardcoded in two places.